sample/src/main/res/layout/wallet_activity.xml

```diff
@@ -48,43 +48,83 @@
                 style="@style/KinButtonStyle.TestNet"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_gravity="right|center"
+                android:layout_gravity="right|top"
                 android:text="@string/refresh"/>
 
             <LinearLayout
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 android:orientation="vertical">
-
                 <LinearLayout
                     android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:orientation="horizontal">
+                    android:layout_height="match_parent"
+                    android:orientation="vertical">
+
+                    <LinearLayout
```

Comment:
I know this is a sample app but nesting of 3 linear layouts? 
consider changing to constraint layout

Reply:
you're right, but I really don't see a point investing effort in this, UI performance is not relevant, nor it has a great impact as we don't have animations or dynamic data here (list)

Reply:
your call but 
1. how much effort would it take?
2. this is open source code - do you not want to give a good example ?